.swiftlint.yml

```diff
@@ -10,6 +10,7 @@ excluded:
   - vendor
   - UISDKdocumentation
   - Tests
+  - Sources/StreamChatUI/Deprecations.swift
```

Comment:
Instead of completely excluding this file from swiftlint, lets just disable rules on a file level.

Reply:
Yeah, we can do this. But I was thinking do we even want deprecations to be covered, especially given this file will be removed in the future

Reply:
+1 on file level

Reply:
Okay, made this change :]